workflow-templates/lint-eslint.yml

```diff
@@ -31,10 +19,31 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  changes:
```

Comment:
```
outputs:
      src: ${{ steps.filter.outputs.src }}
```

Note: filter is the id for the step. I'm fine with changes but used a different id to not confuse it with the job id which is also changes. The outputs section is necessary if we want to use the filter result in a different job within the same workflow. Within the same job it's possible to access the data by the step id.

Reply:
Ah, you just changed that for the spreed pr ;)